apps/system/js/app_window.js

```diff
@@ -223,6 +223,12 @@
     this.getTopMostWindow()._setActive(enable);
   };
 
+  AppWindow.prototype.inScope = function(scope) {
+    var url = new URL(this.url);
```

Comment:
I think this.url needs to be this.config.url (don't ask me why, but .url doesn't get updated)